isovar/variant_sequences.py

```diff
@@ -0,0 +1,254 @@
+# Copyright (c) 2016. Mount Sinai School of Medicine
```

Comment:
There are actually a lot of small changes, I didn't realize the file rename would mask those.